tests/conftest.py

```
@@ -34,4 +56,114 @@
+
+    class _StubFernet:
+        def __init__(self, key):  # pragma: no cover - stub for tests
+            self._key = key
+
+        def encrypt(self, data):
+            payload = data if isinstance(data, bytes) else data.encode()
+            return b"stub:" + payload
+
+        def decrypt(self, token):
+            if not token.startswith(b"stub:"):
+                raise Exception("Invalid token")
```

Ruff (0.14.1)

Create your own exception

(TRY002)